doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+          )
+
+    # coordinate as astropy.units, passing RA in degrees
+    from astropy import units as u
+
+    await tcs.slew_icrs(
+              ra=300.0 * u.deg, dec=-80.0, rot_sky=0.0, target_name="Test target"
+          )
+
+    # coordinate as astropy.Angle
+    from astropy.coordinates import Angle
+
+    await tcs.slew_icrs(
+        ra=Angle(20., unit=u.hourangle),
+        dec=Angle(-80., unit=u.deg),
+        rot_sky=0.,
+        target_name="Test target"
+    )
+
+    # coordinate as astropy.Angle, passing RA in degrees
+    from astropy.coordinates import Angle
+
+    await tcs.slew_icrs(
+        ra=Angle(300., unit=u.deg),
+        dec=Angle(-80., unit=u.deg),
+        rot_sky=0.,
+        target_name="Test target"
+    )
+
+The :py:meth:`slew_icrs <lsst.ts.observatory.control.BaseTCS.slew_icrs>` also implements a couple different rotator positioning strategies.
+The most common strategy is to use ``rot_sky``, also known as position angle (PA), the angle between north direction and the bore-sight y-axis, measured in the eastward direction.
+By default ``rot_sky=0.`` and it can be changed by passing in the desired value;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_sky=90., target_name="Test target"
+          )
+
+Users also have the option to select a physical angle for the rotator.
+For instance, if you are trying to keep the Rotator close to a particular physical range (due to some hardware limitation or observational strategy), use ``rot_phys_sky`` instead;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_phys_sky=20., target_name="Test target"
+          )
+
+This will cause the rotator to be positioned close to the physical (e.g. encoder) angle of ``20.`` degrees.
+Not that this angle is defined at the start of the slew, and the telescope will resume tracking normally, so the rotator will be moving to de-rotate the field.
+
+If instead, you need the rotator to remain fixed at a set position but the telescope must track (e.g. for filter changes on the main telescope), use the ``rot_phys`` option.
+
+.. code:: python
+
+    # WARNING: The telescope will track the alt/az axis but the rotator will
+    # be kept fixed in physical position 0. degrees.
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_phys=0., target_name="Test target"
+          )
+
+When conducting spectroscopy (e.g. with the Auxiliary Telescope) it is useful to be able to position the field in terms of the parallactic angle.
+For that, one can use the ``rot_par`` parameter;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_par=0., target_name="Test target"
+          )
+
+Although ``rot_par=0.`` is the most commonly used value, the user is free to select any angle.
+
+In case the user selects an angle that would cause the rotator to slew to a position outside the valid range, the task will fail and raise an exception.
```

Comment:
"In case the user demands an angle outside the valid range, the task will fail and raise an exception and not slew to the demanded position"